dev/MRTCore/azure-pipelines.yml

```diff
@@ -251,14 +251,26 @@ jobs:
 # START of copying files into their final package location.
 
   - task: CopyFiles@2
-    displayName: 'copy winmd'
+    displayName: 'copy winmd into uap10.0 folder'
     inputs:
       SourceFolder: '$(Build.ArtifactStagingDirectory)\mrt_raw\mrtcore_binaries\lib\anycpu'
       Contents: |
         Microsoft.ApplicationModel.Resources.winmd
       TargetFolder: '$(Build.ArtifactStagingDirectory)\fullnuget\lib\uap10.0\'
       flattenFolders: true
 
+  # Needed so that the NuGet package can be installed in a win32 app. It avoids this error: "You are trying to
```

Comment:
I understand that this winmd is not needed other than to avoid this error... Ideally, we can avoid this error without adding a copy of the winmd, but I don't know how to do that / if that is possible. The error is explicit...